cmd/revad/svcs/httpsvcs/ocdavsvc/get.go

```diff
@@ -92,6 +95,15 @@ func (s *svc) doGet(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	//TODO: make header / auth configurable, check if token is available before doing stat requests
+	tkn, ok := token.ContextGetToken(ctx)
+	if !ok {
+		logger.Error(ctx, errors.New("could not read token from context"))
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+	httpReq.Header.Set("X-Access-Token", tkn)
```

Comment:
The up and download requests need to be authenticated, otherwise the storage driver cannot extract the userid. For now, I hardcoded the example header. Need to think if oidc or basic auth makes sense here. AFAICT only the internal token is used.
@labkode Maybe implement a dummy token writer and credential_strategy as they are not really needed? Or rather ... they should not be available ... then again maybe you want to just do basic auth for up and download ...

Reply:
@butonic I'm refactoring the token manager bussiness as it is used in many places, so it makes sense to inject it in the context